Assets/Scripts/KuriTextManager.cs

```diff
@@ -52,6 +57,29 @@ public enum PRIORITY {
             audioSource.volume = 0.05f;
         }
 
+        public void ToggleGlow()
+        { //toggles glow on and off by changing material, helper method for chest lights
+            if (isGlowing)
```

Comment:
you can probably get rid of `isGlowing` as a variable and instead make it a function that depends on the meshrender's state. So instead of keeping tracking of the state in 2 places, you only do it in one. So a function such as:

```C#
public bool IsGlowing(){
    return lights[0].material == ledOnMaterial;
}
```